### ISSUE TITLE
Dark airplane with model shader full on

### ISSUE THREAD

**Issue by gilbertohasnofb** [Juanvvc/c172p-detailed#324]
Sometimes when there are some clouds the airplane model looks extremely dark when the model shader is full on. Here are some images taken in a spot where the clouds should **not** be casting in shadows in the airplane (last shots shows that there are no cloud shadows in this area):

http://s29.postimg.org/g8fhbhuav/model_shader.jpg

**Comment by tigert** [comment]
I have seen this with many aircraft like cessna 337 for example. I have a laptop which uses the intel opengl driver. Maybe check if this happens with other aircraft as well?

**Comment by wkitty42** [comment]
it is possible the clouds are outside the LOD range for rendering? i've seen this when i turned on cloud shadows... it could be a bug in fg though...

**Comment by gilbertohasnofb** [comment]
@tigert It happens with the Cessna 337 to me as well, but not nearly as bad as in the image I posted above (that is, it doesn't get that much darker). Another example is the Cap10B, which gets only slightly darker but not much at all. So the possibilities are:

- this is caused by some drivers or some GPUs (although unlikely since we are on different vendors: I am using the ATI opengl driver)
- this is caused by the model shader itself in FG (which doesn't explain why some planes work better)
- this is caused by the way some people implement that shader in some aircraft

If it's the last option, then we can try to fix ourselves. If it's one of the first two options, then we must report it to the devs.

**Comment by gilbertohasnofb** [comment]
@wkitty42 

> it is possible the clouds are outside the LOD range for rendering? i've seen this when i turned on cloud shadows... 

I will test that.

**Comment by gilbertohasnofb** [comment]
@wkitty42 

Okay, so indeed this seems to be related to cloud shadows. I believe the shadows are not very precise and therefore the plane looks dark sometimes even though the sun can be directed seen. Also, it's in  some specific weather conditions that this effects gets particularly strong (such as in the screen shots I posted above).

Anyway, so probably this bug isn't related to us at all. Closing it.
